bentoml/_internal/bento/bento.py

```diff
@@ -279,8 +265,11 @@ def from_yaml_file(cls, stream: t.IO[t.Any]):
             logger.error(exc)
             raise
 
+        yaml_content["tag"] = Tag(yaml_content["name"], yaml_content["version"])
+        del yaml_content["name"]
+        del yaml_content["version"]
+        yaml_content["creation_time"] = datetime.fromisoformat(yaml_content["creation_time"])
         bento_info = cls(**yaml_content)
```

Comment:
should we catch the `TypeError` for unexpected keyword argument here and give user a message about unexpected content in `bento.yaml` ?